convert_coverage.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+docker cp backend:/root/core/.coverage .coverage
```

Comment:
will the coverage work with parallel testers? As for example the tester of the autoconf PR?

Reply:
yes codecov merges them per commit